src/utils.ts

```diff
@@ -10,6 +11,10 @@ export interface Options {
 }
 
 export default class Utils {
+  static LOADER_CREDENTIAL_KEYS = {
+    'AWS': ['AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY']
+  };
```

Comment:
We are hardcoding loader specific data here - can we make this inside a loader - maybe a function like getBlackListedKeys which each loader can then fill up?

Reply:
@innoavator I didn't do that because Service to Loader mapping isn't one to one. AWS for example maps to Secrets, SSM and S3. There isn't an abstraction for AWS